tests/generic_config_updater/test_monitor_config.py

```diff
@@ -0,0 +1,214 @@
+    EVERFLOW_DSCP_TEST  MIRROR_DSCP  Ethernet0  EVERFLOW_DSCP_TEST  ingress
+                                     ...
+    
+    admin@vlab-01:~$ show acl rule EVERFLOW_DSCP_TEST RULE_1
+    Table               Rule      Priority  Action                                    Match
+    ------------------  ------  ----------  ----------------------------------------  -------
+    EVERFLOW_DSCP_TEST  RULE_1        9999  MIRROR INGRESS: mirror_session_dscp_test  DSCP: 5
+
+    admin@vlab-01:~/everflow$ show policer everflow_static_policer
+    Name                     Type    Mode         CIR       CBS
+    -----------------------  ------  ------  --------  --------
+    everflow_policer_test    bytes   sr_tcm  12500000  12500000
+
+    admin@vlab-01:~$ show mirror_session mirror_session_dscp_test
+    ERSPAN Sessions
+    Name                      Status    SRC IP    DST IP    GRE      DSCP    TTL  Queue    Policer                  Monitor Port    SRC Port    Direction
+    ------------------------  --------  --------  --------  -----  ------  -----  -------  -----------------------  --------------  ----------  -----------
+    mirror_session_dscp_test  active    1.1.1.1   2.2.2.2               5     32           everflow_policer_test
+    ...
+    """
+    table = duthost.shell("show acl table {}".format(MONITOR_CONFIG_ACL_TABLE))
+    expect_res_success(duthost, table, [MONITOR_CONFIG_ACL_TABLE], [])
+
+    rule = duthost.shell("show acl rule {} {}".format(MONITOR_CONFIG_ACL_TABLE, MONITOR_CONFIG_ACL_RULE))
+    expect_res_success(duthost, rule, [
+        MONITOR_CONFIG_ACL_TABLE, MONITOR_CONFIG_ACL_RULE, MONITOR_CONFIG_MIRROR_SESSION], [])
+
+    policer = duthost.shell("show policer {}".format(MONITOR_CONFIG_POLICER))
+    expect_res_success(duthost, policer, [MONITOR_CONFIG_POLICER], [])
+
+    mirror_session = duthost.shell("show mirror_session {}".format(MONITOR_CONFIG_MIRROR_SESSION))
+    expect_res_success(duthost, mirror_session, [
+        MONITOR_CONFIG_MIRROR_SESSION, MONITOR_CONFIG_POLICER], [])
+
+def verify_no_monitor_config(duthost):
+    """
+    Clean up monitor config in ACL_TABLE, ACL_RULE, POLICER, MIRROR_SESSION
+    """
+    table = duthost.shell("show acl table {}".format(MONITOR_CONFIG_ACL_TABLE))
+    expect_res_success(duthost, table, [], [MONITOR_CONFIG_ACL_TABLE])
+
+    rule = duthost.shell("show acl rule {} {}".format(MONITOR_CONFIG_ACL_TABLE, MONITOR_CONFIG_ACL_RULE))
+    expect_res_success(duthost, rule, [], [
+        MONITOR_CONFIG_ACL_TABLE, MONITOR_CONFIG_ACL_RULE, MONITOR_CONFIG_MIRROR_SESSION])
+
+    policer = duthost.shell("show policer {}".format(MONITOR_CONFIG_POLICER))
+    expect_res_success(duthost, policer, [], [MONITOR_CONFIG_POLICER])
+    
+    mirror_session = duthost.shell("show mirror_session {}".format(MONITOR_CONFIG_MIRROR_SESSION))
+    expect_res_success(duthost, mirror_session, [], [
+        MONITOR_CONFIG_MIRROR_SESSION, MONITOR_CONFIG_POLICER])
+
```

Comment:
Same above. It would be better if we are able to do more check besides CLI level.